src/components/Content/Content.tsx

```diff
@@ -1,30 +1,51 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import styles from './content.module.scss';
+import Card from '../Card/Card';
+import { TMovieCard } from '../../types/TypeMovieDB';
+import { getMovies } from '../../services/movie';
+import Loading from '../Loading/Loading';
+
+const Content: React.FC = () => {
+  const [movies, setMovies] = useState<TMovieCard[]>([]);
+  const [loading, setLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    setLoading(true);
+    setTimeout(() => {
+      getMovies().then((res) => {
+        setLoading(false);
+        setMovies(res);
+      });
+    }, 2000);
+  }, []);
 
-const Content = () => {
   return (
     <div className={styles.container}>
       <aside className={styles.asideContent}>
         <ul>
-          <li>All</li>
-          <li>Türkçe</li>
-          <li>Altyazı</li>
-          <li>Aksiyon</li>
-          <li>Bilim Kurgu</li>
-          <li>Fantazi</li>
+          <li>Trend</li>
+          <li>Popüler</li>
+          <li>2023 Filmleri</li>
+          <li>2022 Filmleri</li>
+          <li>2021 Filmleri</li>
+          <li>2020 Filmleri</li>
         </ul>
       </aside>
       <main className={styles.contentMain}>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
-        <div className={styles.card}>test</div>
+        {loading ? (
+          <Loading />
+        ) : (
+          movies.map((movie: TMovieCard) => {
```

Comment:
```suggestion
movies.map((movie: TMovieCard) => {
            return (
```

yerine 

```suggestion
movies.map((movie: TMovieCard) => (
```

şeklinde de kullanabilirsin. Zorun değil

Reply:
return kaldırıldı, () ile return elde edildi.